app/src/main/java/xyz/klinker/blur/launcher3/Launcher.java

```diff
@@ -502,6 +511,15 @@ protected void onCreate(Bundle savedInstanceState) {
         }
     }
 
+  public void showGlobalSearch() {
+        startSearch(null, false, null, true);
+    }
+
+    public void g(){
```

Comment:
Lets rename this method to something more applicable. It seems to be trying to show the calendar, so maybe `showCalendar` instead?

This actually crashes on my phone though. I would recommend using [this](https://developer.android.com/guide/topics/providers/calendar-provider.html#intent-view). It will open whatever calendar the user has as default, to today's date.